test/includes/scopetest.jl

```diff
@@ -0,0 +1,5 @@
+@testset "Scoping Tests" begin
+    @test_throws LoadError @nbinclude("scoping.ipynb")
```

Comment:
Put the `if VERSION >= v"1.0.0"` test on this line, but still do the `softscope=true` test on earlier versions.

Reply:
Done.